.travis.yml

```diff
@@ -21,7 +21,7 @@ jdk:
   - oraclejdk8
 addons:
   sonarcloud:
-    organization: metafacture
+    organization: hbz
```

Comment:
Could this be causing the Travis error?

```
Execution failed for task ':sonarqube'.
> You're not authorized to run analysis. Please contact the project administrator.
```

https://travis-ci.org/hbz/metafacture-core/builds/507888452

Reply:
On the contrary, I think this is why travis build `#124`is now running correctly.
As I understand it it is only configured to build the master(which is the `#124`).

>  Sonarqube analysis cannot be made when building pull requests because secure
> variables are not available in such builds. To prevent build  failures, the
>sonarqube target is only invoked if not building a pull request.

(https://github.com/hbz/metafacture-core/blob/980-fixUnstoppedThreads/travis/sonarqube.sh#L19)

Why travis is still trying to build this build request I don't get.

I merge and we can see what we will do about this.

Reply:
Uh ."Chances are that the SonarCloud project key inferred from your Gradle configuration is already used by someone else (currently, project keys are uniq across organizations). And the tutorial does not take into account this case." 

Fixed with
 https://github.com/hbz/metafacture-core/commit/6becf02b33d10f1471764afd3808a0e6670bae6b

It's not really nice because this shouldn't go into a PR against the metafacture/metafacture-core origin.